### PR TITLE
Package doi2bib.0.1.4

### PR DESCRIPTION
**Pull request by mseri**
### `doi2bib.0.1.4`
Little CLI tool to get bibtex entries out of doi or arxiv ids

---
* Homepage: https://github.com/mseri/doi2bib
* Source repo: git+https://github.com/mseri/doi2bib.git
* Bug tracker: https://github.com/mseri/doi2bib/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

**Comment by mseri** [comment]
Please don't merge. I am abusing the CI to see the minimum ocaml version required :P

**Comment by avsm** [comment]
If you want to try something a little unreleased, then `ocaml-ci-local` from https://github.com/ocurrent/ocaml-ci will do the solve locally on your project and run a webserver on localhost:8080 that'll show you the minimum OCaml version along with CI results :-)

**Comment by mseri** [comment]
Oh, neat! Thanks!!

File: packages/doi2bib/doi2bib.0.1.4/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "Little CLI tool to get bibtex entries out of doi or arxiv ids"
+maintainer: "marcello.seri@gmail.com"
+authors: "Marcello Seri"
+license: "MIT"
+homepage: "https://github.com/mseri/doi2bib"
+bug-reports: "https://github.com/mseri/doi2bib/issues"
+depends: [
```
Comment (mseri) [inline packages/doi2bib/doi2bib.0.1.4/opam:8]:
```suggestion
depends: [
  "ocaml" {>= "4.08"}
```

**Pull request merged**